### ISSUE TITLE
replace yargs with commander

### ISSUE THREAD

**Issue by MadRabbit** [Prismatik/redbeard#34]
yargs seems pretty good, but [commander](https://www.npmjs.com/package/commander) is so much better. 

it has a nice and clean declarative API for the CLIs, it automatically builds great help pages, supports sub-commands, handles input validation and so on. and it's a goto tool for many devs when they want to build a CLI in node

**Comment by nwinch** [comment]
Yep, looks good.  `yargs` seems to be working fine ATM, so let's mark this as low until it might be something we need.
